src/ralph/settings/base.py

```diff
@@ -167,3 +168,7 @@
     'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.LimitOffsetPagination',  # noqa
     'PAGE_SIZE': 10,
 }
+
+
+IMEI_UNTIL_2003 = re.compile(r'^\d{6} *\d{2} *\d{6} *\d$')
```

Comment:
IMEI_VALIDATORS + dependency injection + allegro custom settings(not settings.py) + later etcd should do the job